addons/l10n_ro_edi/i18n/l10n_ro_edi.pot

```diff
@@ -0,0 +1,56 @@
+# Translation of Odoo Server.
```

Comment:
useless in master, to remove

Reply:
Not for localization, also he adds constraint i think it's worth to have a po

Reply:
For me it needs to be there